packages/app-store/office365video/api/callback.ts

```diff
@@ -65,14 +65,39 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   responseBody.expiry_date = Math.round(+new Date() / 1000 + responseBody.expires_in); // set expiry date in seconds
   delete responseBody.expires_in;
 
-  await prisma.credential.create({
-    data: {
-      type: "office365_video",
-      key: responseBody,
-      userId: req.session?.user.id,
-      appId: "msteams",
-    },
-  });
+  const userId = req.session?.user.id;
+  if (userId) {
```

Comment:
Might be useful to apply the pattern we conveyed as a best practice: [Prefer early returns](https://app.gitbook.com/o/6snd8PyPYMhg0wUw6CeQ/s/VXRprBTuMlihk37NQgUU/engineering/best-practices/prefer-early-returns)

Reply:
Agreed

Reply:
solved